doc/source/guidelines/doc_practices.rst

```diff
@@ -150,8 +150,7 @@ Regarding the documentation dedicated to a specific feature of the product or an
 
 For instance: `cartpole <https://cartpole.mapdl.docs.pyansys.com>`_
```

Comment:
This link is broken too. I am not sure if this should point to https://pyansys.github.io/ml-rl-cartpole/. Pinging here @germa89 just to provide some insight on this.

Reply:
Indeed, the link is broken, Let me see if I can find a good example for this .